modules/tensor_mechanics/src/materials/ComputeMultiPlasticityStress.C

```diff
@@ -77,6 +77,9 @@ ComputeMultiPlasticityStress::ComputeMultiPlasticityStress(const InputParameters
     _my_elasticity_tensor(RankFourTensor()),
     _my_strain_increment(RankTwoTensor())
 {
+  if (_epp_tol <= 0)
+    mooseError("ComputeMultiPlasticityStress: ep_plastic_tolerance must be positive");
+
```

Comment:
OK, then we'll need to engineer that into the rangeCheckedParam call somehow. I'll open a ticket so we can think about it.